google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -243,7 +245,12 @@ public void setArtifact(@Nullable Artifact artifact) {
 
   @Override
   public String getHost() {
-    return settings.getHost();
+    String host = settings.getAdminHost();
+    if (StringUtil.isEmpty(host)) {
```

Comment:
Is this in case other consumers want to use `getHost()`?

`AppEngineServerModel.getDefaultUrlForBrowser()` already supports empty host values.